adafruit_tlv320.py

```diff
@@ -15,6 +15,25 @@
 
 **Hardware:**
 
+* The TLV320DAC chip has moderately complex onboard audio filtering, routing,
+  and amplification capability. Each of the signal chains for speaker, headphone
+  left, and headphone right start with the DAC, then they go through a mixer
+  stage, an analog volume (attenuation) stage, and finally an analog amplifier
+  stage. Parameters for each stage of each signal chain can be separately set
+  with different properties.
+
+* To understand how the different audio stages (DAC, volume, amplifier gain)
+  relate to each other, it can help to look at the Functional Block Diagram in
+  the TLV320DAC3100 datasheet:
+  https://learn.adafruit.com/adafruit-tlv320dac3100-i2s-dac/downloads
+
+* **CAUTION**: The TLV320 speaker amplifier has enough power to easily burn out
+  small 1W speakers if you max out the volume and gain settings. To be safe,
+  start with lower levels for ``speaker_volume`` and ``speaker_gain``, then work
+  your way up to find a comfortable listening level. Similarly, for the
+  headphone output, start low with ``headphone_volume``,
+  ``headphone_left_gain``\, and ``headphone_right_gain``\, then increase as
```

Comment:
Why are the `,`s backslash-escaped here and elsewhere?

Reply:
I have not seen this myself, hmm. Do you mean the closing "``" does not close the opening "``" if followed by punctuation? There are a lot of examples of a closing "``" followed by a comma in the libraries that don't exhibit a problem. Could you give a specific example?

Reply:
yes exactly. In the code you highlighted above, it was rendering as (approximately, working from memory here):
```headphone_left_gain`, and ``headphone_right_gain```

If this is an issue I can just remove all such references in the documentation. That stuff is less important than I'd thought originally. Now that the `speaker_output = True` and `headphone_output = True` stuff is working well, those are definitely the properties that people should use first. Those combined with `dac_volume` should handle most of what people would need.

Reply:
If it makes any difference, I've been using sphinx-build version 8.0.2